coreTable/OwidTable.test.ts

```diff
@@ -507,6 +507,95 @@ describe("time domain", () => {
     })
 })
 
+describe("linear interpolation", () => {
+    const table = new OwidTable(
+        `gdp,year,entityName,entityId,entityCode
+10,2000,france,1,
+0,2001,france,1,
+,2002,france,1,
+,2003,france,1,
+8,2005,france,1,
+,2006,france,1,
+2,2000,uk,2,
+3,2004,uk,2,`,
+        [
+            { slug: "gdp", type: ColumnTypeNames.Numeric },
+            { slug: "year", type: ColumnTypeNames.Year },
+        ]
+    )
+
+    it("applies interpolation", () => {
+        const interpolatedTable = table.interpolateColumnsLinearly("gdp")
+
+        expect(interpolatedTable.rows).toEqual(
```

Comment:
Nit: for this one I think a better assertion would be:

expect(interpolatedTable.get("gdp").allValues.toEqual([10, 0, ...])

Reply:
That way you can ensure all values are correct and it would be easier for a reader to look at the CSV and then the array and understand the test.

Reply:
Ah that's nicer, thanks!